core/tests/unit/ldml/keyboards/fr-t-k0-azerty.xml

```diff
@@ -123,28 +124,28 @@
 		<layer id="base">
 			<row keys="a z e r t y u i o p" />
 			<row keys="q s d f g h j k l m" />
-			<row keys="shift gap w x c v b n gap bksp" />
-			<row keys="numeric extra space enter" /> <!-- The default "space" definition includes `stretch="true"` -->
+			<row keys="shift gap w x c v b n gap" /> <!--TODO:  + bksp -->
+			<row keys="numeric extra space enter" />
 		</layer>
 
 		<layer id="shift">
 			<row keys="A Z E R T Y U I O P" />
 			<row keys="Q S D F G H J K L M" />
-			<row keys="base W X C V B N bksp" />
+			<row keys="base W X C V B N" />  <!--TODO:  + bksp -->
 			<row keys="numeric extra space enter" />
 		</layer>
 
 		<layer id="numeric">
 			<row keys="1 2 3 4 5 6 7 8 9 0" />
-			<row keys="hyphen slash colon semi-colon open-paren close-paren dollar amp at quote" />
-			<row keys="symbol period comma question bang quote bksp" />
+			<row keys="hyphen slash colon semi-colon open-paren close-paren dollar amp at double-quote" />
+			<row keys="symbol period comma question bang double-quote" /> <!--TODO:  + bksp -->
```

Comment:
Why is double-quote here twice?

Reply:
that's in the upstream in ldml. I plan to rename it soon-ish as the 'test keyboard', to make way for   https://unicode-org.atlassian.net/browse/CLDR-12026  as the actual azerty.